packages/data-table/DataTable.stories.tsx

```diff
@@ -10,23 +10,24 @@ export default {
   },
 } as Meta
 
-export const Base: Story<DataTableProps & Pick<DataTableRowProps, 'loading'>> =
-  (props) => {
-    const { loading, ...rest } = props
+export const Base: Story<
+  DataTableProps & Pick<DataTableRowProps, 'loading'>
+> = (props) => {
+  const { loading, ...rest } = props
```

Comment:
```suggestion
> = ({ loading, ...rest }) => {
```

Reply:
This one contains changes from the previous PR